.github/workflows/clang-format.yml

```diff
@@ -0,0 +1,20 @@
+name: Style Check
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  clang-format:
+    name: Clang-Format
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - uses: DoozyX/clang-format-lint-action@v0.12
+      with:
+        source: '.'
+        exclude: './libs'
```

Comment:
We should also exclude `windows/nsProcess/` which is third-party code, I think. It seems like we forgot doing that when doing the first `clang-format` run.
@ann0see As you added that code -- what do you think? I think I'd prefer leaving it untouched. That would mean it should be ignored here and the formatting should be reverted there...?

Reply:
Yes. That’s 3rd party code. I don’t think modifying it does any harm but of course, we can exclude it.

Reply:
I'm mainly worrying about if we ever had to update it. Diffs would look strange at first glance (although this could be fixed by formatting the new code as well).

Reply:
Yes, we should revert it -- I thought I'd caught that at the time but there was so much happening...

Reply:
I have created a separate PR for that: https://github.com/jamulussoftware/jamulus/pull/1751